scenarios/test/nodes_net_consistency.yml

```diff
@@ -0,0 +1,71 @@
+# This scenario simulates a network with the various number of nodes.
+# It starts and stopes nodes at different times.
+# It makes sure that many nodes run till the end of the scenario
+# to allow for the consistency check to see many nodes.
+# The nodes include validators and non-validator nodes.
+# It deploys one application and sends a decent number of transactions.
+
+# The name of the scenario
+name: Network Consistency
+
+# The duration of the scenario's runtime, in seconds.
+duration: 240
+
+# The number of validator nodes in the network.
+num_validators: 2
+
+# The network scenario to exercise. 
+nodes:
+  # Change the number of nodes during the scenario.
+  - name: validator-A
+    start: 20
+    end: 120
+    instances: 2
+    features: [ validator ]
```

Comment:
Specifying that a node is a validator is currently using this syntax:
```
nodes:
  - name: validator-A
    client:
      type: validator
    # features: [ validator ] 
```

This was changed so that we could also specify the client version as well.

Since this is no longer targeted, an alternative to change back this syntax to `features: [ validator ]`.
If you feel strongly about reverting, let me know I will make an additional revert PR.

Reply:
ok updated. I think this configuration has also a big room for improvements: it is not very intuitive, why `validator` is a feature of sub-element `client`, while there is a property `features` - which is not validated when parsed and thus allows for mistakes.  I think this is also artifact of many features that were not finished but are already merged